In [6]:
import scipy.stats as stats
from scipy.stats import norm

3.	1.25% Auditors at a small community bank randomly sample 100 withdrawal transactions made during the week at an ATM machine located near the bank’s main branch. Over the past 2 years, the average withdrawal amount has been $50 with a standard deviation of $40. Since audit investigations are typically expensive, the auditors decide to not initiate further investigations if the mean transaction amount of the sample is between $45 and $55. What is the probability that in any given week, there will be an investigation?

In [20]:
# Finding z score for 45 and 55 values 
# Formula = z=(s_mean-P_mean)/(p_SD/sqrt(n))
z1 = (45-50)/(40/10)
z1

-1.25

In [21]:
z2 = (55-50)/(40/10)
z2

1.25

In [22]:
p = stats.norm.cdf(z2) - stats.norm.cdf(z1)
p

0.7887004526662893

In [24]:
(1-p)*100

21.129954733371072

### Option D) 21.1%

4.	The auditors from the above example would like to maintain the probability of investigation to 5%. Which of the following represents the minimum number transactions that they should sample if they do not want to change the thresholds of 45 and 55? Assume that the sample statistics remain unchanged.

In [95]:
n = 250
mean = 50
std = 40

In [96]:
z1 = (45-mean)/(std/(n)**0.5)

In [97]:
z2 = (55-mean)/(std/(n)**0.5)

In [98]:
p = stats.norm.cdf(z2) - stats.norm.cdf(z1)


In [99]:
1-p

0.0481068278885195

In [100]:
(1-p) * 100

4.81068278885195

Ans : Option D) 250